app/js/components/queryBuilder/queryParamSelector/queryParamSelector.tpl.html

```diff
@@ -24,7 +24,7 @@ <h2>{{$ctrl.currentState()}} Selector</h2>
 		class="tabs__button"
 		ng-click="$ctrl.activeTab = 'search'"
 		ng-class="{'tabs__button--active': $ctrl.activeTab === 'search'}"
-	>Search Results ({{$ctrl.searchResults.length}})</button>
+	>Search Results</button>
```

Comment:
No need to show the number of results now?

Reply:
Yeah, users are more interested in the number of params being added to the query.